src/CONST.js

```diff
@@ -84,6 +84,7 @@ const CONST = {
             BOTTOM_DOCKED: 'bottom_docked',
             POPOVER: 'popover',
             RIGHT_DOCKED: 'right_docked',
+            CONTEXT_MENU: 'context_menu',
```

Comment:
Since the only difference between these two modal types is the value of a couple props, `hasBackdrop` and `coverScreen`, I don't think we need a new modal type here. Just override those props in the places we need to for now. Better yet, just make them both false for the `Popover` modal type by default, which I think makes the most sense?

Reply:
I agree, working on merging the Popover and ContextMenu into Popover